### DISCUSSION TITLE
Configuration file

### DISCUSSION THREAD

**Discussion by NMCoffeeNut** [artisan-roaster-scope/artisan#1540] [General]
I had my artisan set up and the computer died.   I was able to pull out the hard drive and it still works.  It appears when my SIL helped me set this up he did not save the config file.   I have been roasting for over a year and never had an issue.   I copied over the entire directory but I can not find a modified *.aset file.    The computer had been shut down many times so there must be a place artisan holds its working setup file.   Can someone point me to that please.

**Comment by roasterdave** [comment]
You did not say which OS.  I am guessing it is Windows where the working configuration is stored in the registry at
`HKEY_CURRENT_USER\Software\artisan-scope\Artisan`

Depending on the number of customizations you made, it may be easier to just start over and use the **Config>> Machine** menu to create a new configuration.

It is strongly recommended to save your settings to a file using **Help>> Save Settings**.  Backup that file to a safe place, off the computer, so it is always available if needed.

**Reply by NMCoffeeNut** [reply]
Yes, windows, but windows 8.   I will take a look there.   Thanks much
